client/components/tile-grid/README.md

```diff
@@ -0,0 +1,50 @@
+TileGrid (jsx)
+==========
+
+Component used to declare an area for displaying Tiles — it's the main wrapper that takes care of tile alignment and positioning.
+
+#### Props
+
+* `className`: Add your own class to the grid.
+* `isHidden`: Whether the entire grid should be hidden. Default is `false`.
+
+----------
+
+Tile (jsx)
+==========
+
+Component used to display a clickable tile with an image, call to action and description.
+
+#### Props
+
+* `buttonClassName`: Add your own class to the tile button.
+* `buttonLabel`: Text of the button.
+* `className`: Add your own class to the tile.
+* `description`: Description text.
+* `href`: URL that the item leads to upon click.
+* `image`: URL of the image.
+* `onClick`: Function, executed when the user clicks the tile,
+
+----------
+
+#### How to use:
+
+```js
+import Tile from 'components/tile-grid/tile';
+import TileGrid from 'components/tile-grid';
+
+class MyExampleComponent extends Component {
```

Comment:
`PureComponent`, or stateless please 😄 If people are going to copy-paste this, we want them to use best practice.

Reply:
Pure be it 😉  f0ff819c3437bb421d4369dad4102df18bef7afc